src/cache.js

```diff
@@ -266,6 +268,26 @@ class Cache {
       this._cache = {};
       this._saveLastCleared();
   }
+
+  refresh(xmlDoc, refreshedtype) {
```

Comment:
The cache.js file was supposed to be a completely generic cache, i.e. not dependence to XML or Campaign things. I think we should keep it that way, and implement the refresh method outside. For instance in a CacheRefresher object.